CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.13.4]
+- Removes sadd usage in favor of sadd?
```

Comment:
Can we please expand this to explain that this is to address a deprecation warning in preparation for Redis 5?